mad_gui/plot_tools/labels/base_label.py

```diff
@@ -91,6 +94,19 @@ def make_readonly(self):
         self.removable = False
         self.setMovable(False)
 
+    def edit_label_description(self):
```

Comment:
This method is copied from the Region Label class, Correct?

Can you extract it as a function to avoid coping it.

Basically make it a funtion that takes the `parent` and `descriptions` as parameters and returns the selected description.
Within the respective classes it could then be used as:

```python
self.description = edit_label_description(self.descriptions, parent=self.parent.parent)
```